src/ADTypes.jl

```diff
@@ -229,6 +231,20 @@ Chooses [Diffractor.jl](https://github.com/JuliaDiff/Diffractor.jl).
 """
 struct AutoDiffractor <: AbstractADType end
 
+"""
+    AutoFastDifferentiation
+
+Chooses [FastDifferentiation.jl](https://github.com/brianguenter/FastDifferentiation.jl).
+"""
+struct AutoFastDifferentiation <: AbstractSymbolicDifferentiationMode end
```

Comment:
@brianguenter this one. Right now it has no fields, but it would be the right spot to decide "settings" for the FastDifferentiation backend. You can take a look at the other structs to get an idea of the stuff that goes in there